les/api_backend.go

```diff
@@ -51,6 +51,14 @@ func (b *LesApiBackend) ChainConfig() *params.ChainConfig {
 	return b.eth.chainConfig
 }
 
+func (b *LesApiBackend) CacheConfig() *core.CacheConfig {
+	return nil
+}
+
+func (b *LesApiBackend) Coinbase() common.Address {
+	return common.Address{}
```

Comment:
why so

Reply:
In order to let `LesApiBackend` match the `tracers.Backend` interface.